spec/xcpretty/formatters/formatter_spec.rb

```diff
@@ -57,6 +57,11 @@ module XCPretty
 )
     end
 
+    it "formats linker warnings" do
+      @formatter.format_ld_warning("ld: embedded dylibs/frameworks only run on iOS 8 or later").should ==
```

Comment:
Line is too long. [105/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.